packages/components-native/src/InputFieldWrapper/InputFieldWrapper.style.ts

```diff
@@ -34,7 +34,6 @@ export const styles = StyleSheet.create({
   miniLabel: {
     top: 0,
     paddingTop: tokens["space-small"] - tokens["space-smallest"],
-    backgroundColor: tokens["color-surface"],
```

Comment:
When you scroll the input this will overlap the value and the label, won't it?

Reply:
Looks like it does:
![image](https://github.com/GetJobber/atlantis/assets/39704901/098ed637-dc7b-45ed-8ca2-22881d45ef5d)

Reply:
oh thats lovely! 😅